resources/translations/fi.edn

```diff
@@ -223,6 +223,7 @@
                 :field-n "Kenttä %1: %2"
                 :field-title "Kentän otsikko"
                 :field-type "Kentän tyyppi"
+                :info-text "Kenttätiedot, ylimääräinen kuvaus, esimerkkejä (valinnainen)"
```

Comment:
instead of `valinnainen` we use `ei pakollinen` in the others. This should be made to match. Also I think the long name is slightly confusing to users so maybe "Field description", "Kentän kuvaus", "Fält beskrivning"?

Reply:
Addressed